src/components/plots/stackedAreaPlot.ts

```diff
@@ -69,6 +69,13 @@ export module Plots {
       return this;
     }
 
+    protected _updateExtentsForAttr(attr: string) {
+      super._updateExtentsForAttr(attr);
+      if ((attr === "x" || attr === "y") && this._projectorsReady()) {
+        this._updateStackExtents();
+      }
```

Comment:
We lose our type safety though, since we have to do this:
```Typescript
protected _updateExtentsForAttr(attr: string) {
  return (<any> Stacked.prototype)._updateExtentsForAttr.call(this, attr);
}
```
I think the fact that we're running into diamond problems is a strong sign that we've structured the stacked inheritance incorrectly.